src/sif/dynamiccost.cc

```diff
@@ -402,9 +400,6 @@ void ParseBaseCostOptions(const rapidjson::Value& json,
   co->set_disable_hierarchy_pruning(
       rapidjson::get<bool>(json, "/disable_hierarchy_pruning", co->disable_hierarchy_pruning()));
 
-  // top speed
-  JSON_PBF_RANGED_DEFAULT(co, kVehicleSpeedRange, json, "/top_speed", top_speed);
-
   // destination only penalty
```

Comment:
add it to bike as well

Reply:
Actually, I just looked at it again and currently, top speed is not used at all in  bike and pedestrian (in `DynamicCost` it's used in `SpeedPenalty()`, but that's not being called in neither `PedestrianCost` nor `BicycleCost`, so deleting it from `DynamicCost` should be fine.